### ISSUE TITLE
Problems printing with reveal 4.3

### ISSUE THREAD

**Issue by schnatterer** [cloudogu/reveal.js-docker#8]
Some emojis/fontawesome icons are not printed and the background looks different

![image](https://user-images.githubusercontent.com/1824962/162912745-1ee14cc3-aea6-4c7a-a74b-7062370a26b5.png)
![image](https://user-images.githubusercontent.com/1824962/162912896-f343ba40-febf-4cc2-aeaf-af2e542647d7.png)

**Comment by schnatterer** [comment]
One approach: Print with a different tool than headless-chromium.

e.g.
``` 
 docker run fathyb/html2svg 'http://172.17.0.2:8000/?print-pdf' --format pdf -f -W3 -w1024 -h768 > /tmp/out.pdf
```
Looks good, but is printed into one huge page without links.

```
 docker run --rm  -v /tmp:/converted/ arachnysdocker/athenapdf athenapdf -D 2000 -S 'http://172.17.0.2:8000/?print-pdf' > /tmp/a.pdf             
```
This might acutally do it! 😱 I'll try to automate next time I have some more time.

**Comment by schnatterer** [comment]
Another interesting find: The emojis/fontawesome icons printing problem does not occur when printed from dev (nodejs-based) container. Only when using NGINX-based container.

So: switching to `athenapdf` and printing from dev container might solve this issue.

**Comment by schnatterer** [comment]
Fixed in 9f77c1a
